Makefile

```diff
@@ -11,6 +12,9 @@ endif
 # Note in order to install local (faster) yaml: "go get github.com/mikefarah/yaml"
 YAML_CMD:=$(shell which yaml || echo docker run --rm -i calico/yaml)
 
+# Local directories to ignore when running htmlproofer
+HP_IGNORE_LOCAL_DIRS="/v1.5/,/v1.6/,/v2.0/,/v2.1/,/v2.2/,/v2.3/,/v2.4/,/v2.5/,/v2.6/,/v3.0/"
```

Comment:
might be easier to maintain if we invert this and only run htmlproofer on master, 404.html, and the toplevel index.html

Reply:
Oh I see this is not new code, just moved. your call if you want to implement this change or not